crates/orderbook/openapi.yml

```diff
@@ -435,9 +437,14 @@ components:
            Note that for ethflow orders, the valid_to encoded in the smart
            contract is max(uint)
           type: integer
+        isRefunded:
+          description: |
+           Is ETH refunded
+          type: boolean
```

Comment:
AFAIK, we purposefully left this out as we do not want to support it moving forward. Maybe @josojo can comment (as I'm not 100% sure it was this field, or another field).

Reply:
Removed it, thanks!